api/src/utils/shared-api-docs.ts

```diff
@@ -0,0 +1,148 @@
+import { WRITE_ROLES } from '../constants/roles';
+import { projectFundingSourcePostRequestObject } from '../openapi/schemas/project-funding-source';
+
+export const getAttachmentApiDocObject = (basicDescription: string, successDescription: string) => {
+  return {
+    description: basicDescription,
+    tags: ['attachment'],
+    security: [
+      {
+        Bearer: WRITE_ROLES
+      }
+    ],
+    parameters: [
+      {
+        in: 'path',
+        name: 'projectId',
+        schema: {
+          type: 'number'
+        },
+        required: true
+      },
+      {
+        in: 'path',
+        name: 'attachmentId',
+        schema: {
+          type: 'number'
+        },
+        required: true
+      }
+    ],
+    responses: {
+      200: {
+        description: successDescription,
+        content: {
+          'text/plain': {
+            schema: {
+              type: 'number'
+            }
+          }
+        }
+      },
+      401: {
+        $ref: '#/components/responses/401'
+      },
+      default: {
+        $ref: '#/components/responses/default'
+      }
+    }
+  };
+};
+
+export const deleteFundingSourceApiDocObject = (basicDescription: string, successDescription: string) => {
```

Comment:
can we add a test for this later (doesn't have to be in this pr) 
it will be copy paste of the `getAttachmentApiDocObject` test

Reply:
good catch, I made a note to add this to the next PR.